test/manual/amp-ad.yieldbot.amp.html

```diff
@@ -27,7 +27,7 @@
           data-psn="1234"
           data-yb-slot="medrec"
           data-slot="/2476204/medium-rectangle"
-          data-multi-size="300x220,300x200"
+          data-multi-size="300x220,300x210, 300x200,300NNN,300xNNN,"
```

Comment:
That's a good point. Let's leave the test here. 

FYI: We may move the test later, if we see more manual tests from other 3p ad networks.